src/webgpu/api/validation/encoding/queries/begin_end.spec.ts

```diff
@@ -9,15 +9,50 @@ TODO:
     - begin 2, end 2
     - }
     - x= {
-        - render pass + occlusion
         - render pass + pipeline statistics
         - compute pass + pipeline statistics
         - }
-- nesting: test whether it's allowed to nest various types of queries
-  (including writeTimestamp inside begin/endable queries).
 `;
 
 import { makeTestGroup } from '../../../../../common/framework/test_group.js';
 import { ValidationTest } from '../../validation_test.js';
 
 export const g = makeTestGroup(ValidationTest);
+
+g.test('occlusion_query,begin_end')
+  .desc(
+    `
+Tests that begin/end occlusion query on render pass:
+- begin 0
```

Comment:
For this description we should be a bit more explicit, e.g. "begin index 0, begin index 1, end" or something.

(Note, in the notes above, "begin 0, end 1" means "begin no queries and end one query". That's unclear, so best to be clear here.)